internal/task/executor.go

```diff
@@ -68,5 +70,8 @@ func runTaskSafely(ctx context.Context, t Task, resolver file.Resolver, s sbomsy
 		}
 	}()
 
-	return t.Execute(ctx, resolver, s)
+	start := time.Now()
+	res := t.Execute(ctx, resolver, s)
+	log.Debugf("task name: %s time to execute: %v", t.Name(), time.Since(start))
```

Comment:
this is probably better served as a `trace` log level detail

Reply:
fixed